packages/smooth_app/lib/database/robotoff_questions_query.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:async';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/utils/QueryType.dart';
+import 'package:smooth_app/database/product_query.dart';
+
+class RobotoffQuestionsQuery {
+  RobotoffQuestionsQuery(this.barcode);
+  final String barcode;
+
+  Future<List<RobotoffQuestion>> getRobotoffQuestionsForProduct() async {
+    final RobotoffQuestionResult result =
+        await OpenFoodAPIClient.getRobotoffQuestionsForProduct(
+      barcode,
+      ProductQuery.getLanguage().code,
+      count: 3,
+      queryType: QueryType.PROD,
```

Comment:
No it's not `QueryType.PROD`, please leave it blank.
And if you have good reasons to force if to `QueryType.PROD`, please add comments detailing why.